app/views/users/show.html.erb

```diff
@@ -1 +1,21 @@
-<h4>Here is single of user.</h4>
\ No newline at end of file
+<div class = "container">
+    <div class = "user-row">
+        <div class = "photo">
+            <%= image_tag @user.photo %>
+        </div>
+        <div class = "intro-details">
+            <p class = "user-name"><%= @user.name %></p>
+            <p class = "post-counter">Number of posts:<%= @user.posts_counter %></p>
+        </div>
```

Comment:
- [ ] Kindly take note that we use column names to display data from the database, for this reason, `posts_counter` should match the column name. I advise you to correct all the errors and test the app before submitting it for review.